src/Model/Item.php

```diff
@@ -422,4 +1067,659 @@ public static function updateContact($arr) {
 			}
 		}
 	}
+
+	/**
+	 * The purpose of this function is to apply system message length limits to
+	 * imported messages without including any embedded photos in the length
+	 *
+	 * @brief Truncates imported message body string length to max_import_size
+	 * @param string $body
+	 * @return string
+	 */
+	public static function limitBodySize($body)
```

Comment:
This function pertains to BBCode body strings, please move it to `BBCode::truncateBodyText()`

Reply:
Whatever "pertain" means - but the word sounds funny :-) (I will move it, but currently have to do some support via telephone)

Reply:
"belongs to", "is about".

Family technical support?